res_partner_stock_picking/models/stock_picking.py

```diff
@@ -0,0 +1,14 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _pre_action_done_hook(self):
+        for picking in self:
+            never_parcial_recepcion = picking.partner_id.create_partial_reception
```

Comment:
Calling this variable "never" is not accurate... it's simply partial reception value for picking contact, we don't know its value at this point. And please check typos for "parcial" and "recepcion" (at least they are mistakes, in english,...)